microcloud/test/includes/microcloud.sh

```diff
@@ -287,7 +287,7 @@ validate_system_lxd_ovn() {
 
   # Check that the created UPLINK network has the right DNS servers.
   if [ -n "${dns_namesersers}" ] ; then
-    dns_addresses=$(lxc exec "${name}" -- sh -c "lxc network get UPLINK dns.nameservers")
+    dns_addresses=$(lxc exec "local:${name}" -- sh -c "lxc network get UPLINK dns.nameservers")
```

Comment:
Is this commit specific to this PR or can it be moved into a different one?

Reply:
Removed it.